src/main/java/uk/ac/ebi/atlas/search/JsonGeneSearchController.java

```diff
@@ -83,13 +85,13 @@ public String search(@RequestParam MultiValueMap<String, String> requestParams)
         if (category.equals("q")) {
             geneQuery =
                     species.map(_species -> GeneQuery.create(queryTerm, _species))
-                           .orElseGet(() -> GeneQuery.create(queryTerm));
+                            .orElseGet(() -> GeneQuery.create(queryTerm));
         } else {
             geneQuery =
                     species.map(
                             _species ->
                                     GeneQuery.create(queryTerm, BioentityPropertyName.getByName(category), _species))
-                           .orElseGet(() -> GeneQuery.create(queryTerm, BioentityPropertyName.getByName(category)));
+                            .orElseGet(() -> GeneQuery.create(queryTerm, BioentityPropertyName.getByName(category)));
 
         }
         Optional<ImmutableSet<String>> geneIds = geneIdSearchService.search(geneQuery);
```

Comment:
Java 11 includes [`Optional::isEmpty`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Optional.html#isEmpty()), could you change it below?